### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by scottmulligan**
- Email and Phone number did not appear in xProfile (Details tab)
- Home installation form had fields in wrong order
- Removed HTML junk content from Home Installation hero content

### CODE REVIEW

File: src/Foundation/Accounts/code/Services/ContactFacetService.cs [hunk 1]
```diff
@@ -200,18 +200,7 @@ private bool SetEmail(ContactFacetData data, XConnect.Contact contact, XConnectC
                 return false;
             }
             var emails = contact.Emails();
-            if (emails == null)
```
Comment (jeanfrancoislarente) [inline src/Foundation/Accounts/code/Services/ContactFacetService.cs]: Hey @scottmulligan 
Looks like you're setting e-mails to contact.Emails() but immediately overwriting it with new EmailAddressList. Are you now assuming that the contact will never have an existing e-mail?

If the business logic is fine maybe ermove the 
`var emails = contact.Emails();`
and add the declaration to the next line:
`var emails = new EmailAddressList(new EmailAddress(email, true), "Work Email");`

File: src/Foundation/Accounts/code/Services/ContactFacetService.cs [hunk 2]
```diff
@@ -224,18 +213,7 @@ private bool SetPhone(ContactFacetData data, XConnect.Contact contact, XConnectC
                 return false;
             }
             var phoneNumbers = contact.PhoneNumbers();
-            if (phoneNumbers == null)
```
Comment (jeanfrancoislarente) [inline src/Foundation/Accounts/code/Services/ContactFacetService.cs]: Same as above with `var phoneNumbers = contact.PhoneNumbers();`. Redundant?

Comment (scottmulligan) [reply src/Foundation/Accounts/code/Services/ContactFacetService.cs]: Yep, I fully agree. Doesn't make sense. I will just create a new facet and place it in the EmailList or Phone NumberList. Thanks for taking a look.

**Comment by scottmulligan** [comment]
@jeanfrancoislarente - OK, PR updated and tested. Please review and let me know if any issues.

Thanks!

**Pull request merged**